source/dem/dem.cc

```diff
@@ -519,6 +520,13 @@ DEMSolver<dim>::set_insertion_type(const DEMSolverParameters<dim> &parameters)
         std::make_shared<NonUniformInsertion<dim>>(parameters,
                                                    maximum_particle_diameter);
     }
+  else if (parameters.insertion_info.insertion_method ==
+           Parameters::Lagrangian::InsertionInfo::InsertionMethod::list)
+    {
+      insertion_object =
+        std::make_shared<ListInsertion<dim>>(parameters,
+                                             maximum_particle_diameter);
```

Comment:
Why do you need maximum_particle_diameter parameter for this insertion type?

Reply:
We don't, good catch.